From e3c4d699f2baab58abf0c36fc8cfec2c78dda6df Mon Sep 17 00:00:00 2001
From: Andrei Suprun <andrei.suprun.by@gmail.com>
Date: Thu, 1 Apr 2021 18:12:05 +0300
Subject: [PATCH] edit FirstFile.txt

---
 FirstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/FirstFile.txt b/FirstFile.txt
index f00c965..4226016 100644
--- a/FirstFile.txt
+++ b/FirstFile.txt
@@ -1,5 +1,5 @@
-1
-2
+q
+w
 3
 4
 5
-- 
2.24.1.windows.2

